src/components/Toast/README.md

```diff
@@ -377,3 +438,25 @@ Keep the action label short, preferably 1 verb action.
 
 - To present a small amount of content or a menu of actions in a non-blocking overlay, [use the popover component](/components/popover)
 - To communicate a change or condition that needs the merchant’s attention within the context of a page, [use the banner component](/components/feedback-indicators/banner)
+
+  ***
```

Comment:
@sadiesaurus This was the problem. My keyboard has gone awry and this happened. Thanks for pointing it out. 🤦‍♀️

Reply:
This looks great to me!

Reply:
🎉